sdk/rust/nym-sdk/src/bandwidth/client.rs

```diff
@@ -20,9 +19,9 @@ pub type VoucherBlob = Vec<u8>;
 /// [`crate::mixnet::DisconnectedMixnetClient::create_bandwidth_client`] on the associated mixnet
 /// client.
 pub struct BandwidthAcquireClient<'a, St: Storage> {
-    network_details: NymNetworkDetails,
     client: DirectSigningHttpRpcNyxdClient,
     storage: &'a St,
+    client_id: String,
```

Comment:
ugh. i've just noticed this is the private key. does it have to be private and not public? if so. it has to be zeroized so derive the traits

Reply:
alternatively, if you only need references, you can change it to `client_id: Zeroizing<String>`

Reply:
It has to be something private. Otherwise the resulting ecash key pair isn't secret. It doesn't necessarily has to be the private key though